packages/core/content-releases/admin/src/services/release.ts

```diff
@@ -6,18 +6,50 @@ import { axiosBaseQuery } from './axios';
 
 import type { CreateRelease, GetReleases } from '../../../shared/contracts/releases';
 
+export interface GetAllReleasesQueryParams {
+  page?: number;
+  pageSize?: number;
+  filters?: {
+    $and?: Array<{
+      releasedAt?: {
+        $notNull?: boolean | 'true' | 'false';
```

Comment:
we want to accept strings?

Reply:
because in the transformResponse the arguments values are strings and I need them to activate or not a tab (based on filters)